cardano-node/src/Cardano/CLI/Parsers.hs

```diff
@@ -421,53 +466,12 @@ parseTxRelatedValues =
         $ SpendUTxO
             <$> parseProtocol
             <*> (GenesisFile <$> parseGenesisPath)
-            <*> parseGenesisHash
```

Comment:
I would prefer if we keep this argument as otherwise we need to adapt the configuration file if genesis is recreated.

Reply:
See: https://github.com/input-output-hk/cardano-node/pull/540/files#r375498783